src/components/HomePage/Banner.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import { useTheme, makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import PropTypes from 'prop-types';
+
+import Countdown from 'react-countdown';
+
+import SvgImg from '../SvgImg';
+import hothLogo from './hoth7-logo.svg';
+import hothBanner from './hoth-banner.png';
+
+const hothStart = new Date('2021-01-23T09:00:00-07:00');
+
+const useStyles = makeStyles(theme => ({
+	colon: {
+		fontFamily: 'open-sans',
+		color: 'white',
+		fontWeight: 600,
+		fontSize: theme.typography.fontSize * 3
+	},
+	container: {
+		backgroundColor: theme.palette.primary.dark,
+		height: 'auto',
+		[theme.breakpoints.up('md')]: {
+			backgroundImage: `url(${hothBanner})`,
+			backgroundSize: '960px 327px',
+			backgroundRepeat: 'no-repeat',
+			backgroundPosition: 'right bottom'
+		},
+		padding: 25
+	},
+	countdown: {
+		padding: 20
+	},
+	text: {
+		color: 'white',
+		fontWeight: 500
+	},
+	logo: {
+		width: 150,
+		margin: 20,
+		[theme.breakpoints.down('sm')]: {
+			width: 150
+		}
+	},
+	time: {
+		color: 'white',
+		fontWeight: 600,
+		fontSize: theme.typography.fontSize * 3
+	},
+	timeDescD: {
+		'&::after': {
+			content: '"days"',
+			fontWeight: 400,
+			fontSize: theme.typography.fontSize * 0.6
+		}
+	},
+	timeDescH: {
+		'&::after': {
+			content: '"hours"',
+			fontWeight: 400,
+			fontSize: theme.typography.fontSize * 0.6
+		}
+	},
+	timeDescM: {
+		'&::after': {
+			content: '"minutes"',
+			fontWeight: 400,
+			fontSize: theme.typography.fontSize * 0.6
+		}
+	},
+	timeDescS: {
+		'&::after': {
+			content: '"seconds"',
+			fontWeight: 400,
+			fontSize: theme.typography.fontSize * 0.6
+		}
+	}
+}));
+
+function renderInfo(classes) {
+	return (
+		<Grid
+			container
+			direction='column'
+			justify='center'
+			alignItems='center'
+		>
+			<Typography variant='h3' className={classes.text}>Hack on the Hill 7</Typography>
+			<Typography variant='h5' className={classes.text}>Explore. Build. Empower.</Typography>
+			<SvgImg src={hothLogo} className={classes.logo} width={258} height={250} />
+			<Typography variant='h5' className={classes.text}>Feb 23rd, 2021</Typography>
```

Comment:
```suggestion
			<Typography variant='h5' className={classes.text}>Feb 23<sup>rd</sup>, 2021</Typography>
```

Reply:
![image](https://user-images.githubusercontent.com/35614552/84618369-02036a00-aea0-11ea-9e8b-562df795ad3b.png)
I feel like this looks strange, thoughts?